spec/array_inject_spec.rb

```diff
@@ -0,0 +1,7 @@
+require 'array_inject'
+
+describe SubclassArray do 
+	it 'has a working inject method' do 
```

Comment:
Use 2 (not 1) spaces for indentation.<br>Tab detected.<br>Trailing whitespace detected.